openapi/kas-fleet-manager.yaml

```diff
@@ -1004,9 +1004,9 @@ components:
               type: string
             size_id:
               type: string
-            ingress_throughput_per_sec:
+            ingress_per_sec:
               type: string
-            egress_throughput_per_sec:
+            egress_per_sec:
```

Comment:
Are these changes intentional? If so, let's make the UI/CLI team aware of it

Reply:
that's what I understood from the chat. I am happy to reverse it

Reply:
ok. makes sense

Reply:
@pawelpaszki sorry, yes I should have been more clear. The goal is to update the names for the data plane (capacity struct) only. Unfortunately the contract with the UI now uses different names, but revising that should be a separate task.

Reply:
opened new PR here instead of this one: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/946